src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -80,52 +99,99 @@ def add_profile_args(parser):
         required=False,
         help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
     )
-
-
-def add_endpoint_args(parser):
     parser.add_argument(
-        "--url",
+        "--profile-export-file",
         type=str,
-        default="localhost:8001",
         required=False,
-        help="URL of the endpoint to target for benchmarking.",
+        help="Specifies the path that the profile export will be "
+        "generated at. By default, the profile export will not be "
+        "generated.",
     )
     parser.add_argument(
-        "--provider",
+        "--request-rate",
+        type=float,
+        required=False,
+        help="Sets the request rates for load generated by analyzer. ",
+    )
+    parser.add_argument(
+        "--service-kind",
         type=str,
         choices=["triton", "openai"],
+        default="triton",
         required=False,
-        help="Provider format/schema to use for benchmarking.",
+        help="Sets the request rates for load generated by analyzer. "
+        "Describes the kind of service perf_analyzer to "
+        'generate load for. The options are "triton" and '
+        '"openai". The default value is "triton".',
     )
-
-
-def add_dataset_args(parser):
     parser.add_argument(
-        "--dataset",
-        type=str,
-        default="OpenOrca",
-        choices=["OpenOrca", "cnn_dailymail"],
+        "--streaming",
+        action="store_true",
+        required=False,
+        help=f"Enables the use of streaming API. This flag is "
+        "only valid with gRPC protocol. By default, it is set false.",
+    )
+    parser.add_argument(
+        "--sync",
+        action="store_true",
+        required=False,
+        help=f"Enables asynchronous mode in perf_analyzer. "
+        "By default, perf_analyzer will use synchronous API to "
+        "request inference. However, if the model is sequential, "
+        "then default mode is asynchronous. Specify --sync to "
+        "operate sequential models in synchronous mode. In synchronous "
+        "mode, perf_analyzer will start threads equal to the concurrency "
+        "level. Use asynchronous mode to limit the number of threads, yet "
+        "maintain the concurrency.",
+    )
+    parser.add_argument(
+        "--version",
+        action="store_true",
         required=False,
-        help="HuggingFace dataset to use for the benchmark.",
+        help=f"Enables the printing of the current version of perf_analyzer. "
+        "By default, it is set false.",
     )
+
+
+def add_endpoint_args(parser):
     parser.add_argument(
-        "--tokenizer",
+        "--u",
```

Comment:
I would do `-u` / `--url` rather than `--u`

Reply:
Under the hood, PA needs -u. 
I can do that, but I need to update the command before i make the call.

Reply:
`-u` is fine, just saying single line `-` rather than double `--` for the shortened form. The python argparse can accept both short and long form if you specify both `parser.add_argument("-u", "--url", ...)`